pom.xml

```diff
@@ -126,6 +128,11 @@
                 <artifactId>fest-assert</artifactId>
                 <version>${version.fest-assert}</version>
             </dependency>
+            <dependency>
+                <groupId>org.assertj</groupId>
+                <artifactId>assertj-core</artifactId>
```

Comment:
Personally, I would leave it for a separate PR. As Alex has mentioned, the target of this PR is implementing Gradle support, not refactoring the current tests...